localization/english/wc_traits_l_english.yml

```diff
@@ -487,6 +489,15 @@
  trait_dire:0 "Dire"
  trait_dire_character_desc:0 "$trait_dire_desc$"
  trait_dire_desc:0 "A small number of trolls are born larger and more powerful than their brethren. Those blessed with this form are called dire trolls, or berserkers, and they stand at the very top of the troll fighting elite."
- 
+
  TRAIT_FLAG_DESC_return_to_twisting_nether:0 "Can return to [twisting_nether|E]"
- TRAIT_FLAG_DESC_makes_incapable:0 "@warning_icon!#X Makes [wc_incapable|E]#!"
\ No newline at end of file
+ TRAIT_FLAG_DESC_makes_incapable:0 "@warning_icon!#X Makes [wc_incapable|E]#!"
+
+ trait_lich_queen:0 "Lich Queen"
+ trait_lich_king_common_desc:0 "Originally created by Kil'jaeden from the spirit of the orc shaman, Ner'zhul, who he bound within the Helm of Domination. The Lich King's vast, telepathic influence and unimaginable necromantic power allows him to control endless hordes of living dead."
```

Comment:
If there's localization for `trait_lich_queen`, shouldn't there also be a `trait_lich_queen_common_desc`? I'm only seeing the male version here.

Reply:
Changed in bd21345209f4ec7c31efca92d215c3135a6fda3f.